marklogic-data-hub/src/main/resources/ml-modules/root/com.marklogic.hub/lib/hub-entities.xqy

```diff
@@ -187,84 +75,39 @@ declare function hent:wrap-duplicates(
         $item)
 };
 
-declare function hent:dump-search-options($entities as json:array)
+declare %private function hent:fix-options($nodes as node()*)
```

Comment:
I can see that this function replaces "es:" with "*:" in text nodes, but it's not clear why that's a useful thing. Please add a comment.

Reply:
Just saw the comment further down about entity services issue #359. Okay.